temporalcli/commandsmd/commands.md

```diff
@@ -1,221 +1,537 @@
-      * `<option-names>` is `` `--<option-name>` `` and can optionally be followed by ``, `-<short-name>` ``.
-      * `<data-type>` must be one of `bool`, `duration`, `int`, `string`, `string[]`, `string-enum`, `timestamp`, TODO: more
-      * `<short-description>` can be just about anything so long as it doesn't match trailing attributes. Any wrap
-        around to newlines + two-space indention is trimmed to a single space.
-      * `<extra-attributes>` can be:
-        * `Required.` - Marks the option as required.
-        * `Default: <default-value>.` - Sets the default value of the option. No default means zero value of the type.
-        * `Options: <option>, <option>.` - Sets the possible options for a string enum type.
-        * `Env: <env-var>.` - Binds the environment variable to this flag.
-        * ``Alias: `--<alias>`.`` - Sets a flag as an alias of this one. The flag should not be separately defined.
-      * Options should be in order of most commonly used.
-    * Also can have single lines below options that say
-      `Includes options set for [<options-set-name>](#options-set-for-<options-set-link-name>).` which is the equivalent
-      of just having them appended to the bulleted list.
-      * Just because a command may have a couple of similar options with another doesn't mean you _have_ to make a
-        shareable options set. Copy/paste is acceptable.
-* Keep commands in alphabetical order.
-* Commands that have subcommands cannot be run on their own.
-* Keep lines at 120 chars if possible.
-* Add punctuation even at the end of phrases.
+This document is automatically parsed.
+Follow these rules.
+
+IN-HOUSE STYLE
+
+* Wording and grammar.
+  * Run a spell check.
+  * Be clear and concise.
+  * Don't reword the command in the short description. Use distinct supplementary language.
+    * Yes: temporal workflow delete: Remove Workflow Execution
+    * No: temporal workflow delete: Delete Workflow
+  * Re-use and adapt existing wording and phrases wherever possible.
+  * Word command short descriptions as if they began "This command will..."
+    Use sentence casing for the short description.
+  * ID is fully capitalized in text ("the Workflow ID") and Id in [metasyntax](https://en.wikipedia.org/wiki/Metasyntactic_variable) (YourWorkflowId).
+* Avoid parentheticals unless absolutely necessary.
+
+* Wrapping:
+  * Assume a user-visible line length of 80 characters max. `fmt -w 78` can help.
+    * Long descriptions must be pre-wrapped.
+  * When declaring Options follow the wrapping style used elsewhere in this file.
+    Splitting the items into multiple lines improves maintainability with clearer diffs.
+
+* Ordering:
+  * Commands: Use alphabetical order for commands.
+  * Options: Most commonly used options first.
+
+* Punctuation:
+  * With the exception of short descriptions and triple-quoted code-fenced samples, end everything with a period.
```

Comment:
This is 114 characters and violates the 80 char rule that was imposed just above. Having said that, the rule is a bit too limiting IMO.

Reply:
You're not wrong but this is in a comment and cannot be seen or rendered outside source. Since it's not user-facing, I don't think it's going to be a problem. Thank you for calling it out for clarification.